dev/com.ibm.ws.jdbc_fat_derby/publish/servers/com.ibm.ws.jdbc.fat.derby/server.xml

```diff
@@ -48,7 +50,33 @@
       <properties.derby.embedded databaseName="memory:ds4" createDatabase="create"/>
     </dataSource>
     
+    <dataSource id="dsfat5" jndiName="jdbc/dsfat5" type="javax.sql.DataSource" isolationLevel="TRANSACTION_NONE" transactional="false">
+      <jdbcDriver libraryRef="DerbyLib" />
+      <authentication-alias name="derbyAuth"/>
+      <properties.derby.embedded databaseName="memory:ds5" createDatabase="create"/>
+    </dataSource>
+    
+    <!-- dsfat6, dsfat7, dsfat8, dsfat9 are defined via @DataSourceDefinition in JDBCDerbyServlet -->
+    
+    <dataSource id="dsfat10" jndiName="jdbc/dsfat10" type="javax.sql.DataSource" isolationLevel="TRANSACTION_SERIALIZABLE" transactional="false">
+      <jdbcDriver libraryRef="DerbyLib" />
+      <authentication-alias name="derbyAuth"/>
+      <properties.derby.embedded databaseName="memory:ds10" createDatabase="create"/>
+    </dataSource>
+    
+    <dataSource id="dsfat11" jndiName="jdbc/dsfat11" type="javax.sql.DataSource" isolationLevel="TRANSACTION_NONE" transactional="false">
```

Comment:
This seems identical to  dsfat11, is it really needed?  Should also check all the other datasources are really needed, it makes it easier later if we only add as many datasource configs are are actually needed.